2021-05-13 19:19:25,535 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2021-05-13 19:19:25,535 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from D:\instagram\venv\lib\site-packages\instabot\bot\bot.py
2021-05-13 19:19:25,546 - instabot version: 0.117.0 (api_login) - INFO - Not yet logged in starting: PRE-LOGIN FLOW!
2021-05-13 19:19:26,462 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-05-13 19:19:26,462 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-05-13 19:19:26,463 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-05-13 19:19:26,463 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 5 minutes.
2021-05-13 19:24:27,116 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: accounts/get_prefill_candidates/ returned response: <Response [429]>
2021-05-13 19:24:27,116 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message":"Please wait a few minutes before you try again.","status":"fail"}'
2021-05-13 19:24:27,117 - instabot version: 0.117.0 (api) - ERROR - Request returns 429 error!
2021-05-13 19:24:27,117 - instabot version: 0.117.0 (api) - WARNING - That means 'too many requests'. I'll go to sleep for 10 minutes.
